sources/lib/Twig/Extension/ProfilerExtension.php

```diff
@@ -30,9 +30,9 @@ class ProfilerExtension extends \Twig_Extension
      * Extension constructor.
      *
      * @access public
-     * @param  FilesystemLoader $loader
+     * @param  FilesystemLoader|\Twig_Loader_Filesystem $loader
      */
-    public function __construct(FilesystemLoader $loader)
```

Comment:
You can use `\Twig_Loader_Filesystem` for type hinting, `FilesystemLoader` extends its.